test/CurrentHookAddress.t.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {Lockers} from "../src/libraries/Lockers.sol";
+import {IHooks} from "../src/interfaces/IHooks.sol";
+
+contract CurrentHookAddressTest is Test {
+    function test_getCurrentHook() public {
+        assertEq(address(Lockers.getCurrentHook()), address(0));
+    }
+
+    function test_setCurrentHook() public {
+        Lockers.setCurrentHook(IHooks(address(1)));
+        assertEq(address(Lockers.getCurrentHook()), address(1));
+    }
+
+    function test_setCurrentHook_TwiceDoesNotSucceed() public {
+        Lockers.setCurrentHook(IHooks(address(1)));
+        Lockers.setCurrentHook(IHooks(address(2)));
```

Comment:
i think this should be checking the return values?
like if it changed -> true, if it didnt change -> false